sdk/identity/identity/README.md

```diff
@@ -1,56 +1,126 @@
-## Azure Identity client library for Javascript
+## Azure Identity client library for JS
```

Comment:
What was the motivation behind this change from `Javascript` to `JS`?

Reply:
For whatever reason, this is the string that `doc-warden` wants to see in the README:

https://github.com/Azure/azure-sdk-for-js/blob/967179650ab72200e2b2e06745f0e6074af48700/.docsettings.yml#L8-L15

I think that should definitely be changed to say "JavaScript" instead of "JS" at the very least.

Reply:
Weird, event-hubs has it as "Javascript"

Reply:
event-hubs is in the "known issues" exclusion list:

https://github.com/Azure/azure-sdk-for-js/blob/41e716cbcdead0ef15d4ff744ec9a1d39a24b986/.docsettings.yml#L80

We'll need to update all our README.md files to be compliant with the checks and then remove ourselves from this list. That's what I've been doing for core-http and identity PRs